### ISSUE TITLE
Error: Unable to Access Root Module File in TRPC Generator

### ISSUE THREAD

**Issue by DKeken** [KevinEdry/nestjs-trpc#29]
**Description:**

I'm encountering an error when using the `nestjs-trpc` package that prevents the root module file from being accessed during the generation process. This error occurs when attempting to initialize the module with `TRPCGenerator`.

The error message is as follows:

```
Error: Could not access root module file.
    at TRPCGenerator.buildRootImportsMap (...\nestjs-trpc\lib\generators\trpc.generator.ts:197:13)
    at TRPCGenerator.onModuleInit (...\nestjs-trpc\lib\generators\trpc.generator.ts:71:38)
    at MapIterator.iteratee (...\@nestjs\core\hooks\on-module-init.hook.js:22:43)
    ...
```

**Steps to Reproduce:**

1. Download any example from the `nestjs-trpc` repository.
2. Install dependencies and run the example.
3. The error is thrown during the initialization process when generating TRPC schemas or imports.

**Expected Behavior:**

The TRPC generator should correctly access the root module file and generate necessary schemas and imports without errors.

**Environment:**

- Node.js version: 22.6.0
- NestJS version: 10.4.1
- nestjs-trpc version: 1.5.1
- OS: Windows

**Additional Notes:**

It seems like the error is related to the TRPC generator failing to locate the root module file, potentially due to issues with paths or module resolution. Any guidance or suggestions on resolving this would be greatly appreciated!

![image](https://github.com/user-attachments/assets/5bb13b1c-d8eb-456f-a654-3244d684cf5c)

**Comment by DKeken** [comment]
I tried switching to version 1.4.0, and now I'm getting this error...
![image](https://github.com/user-attachments/assets/eedb4ef5-9684-4105-8006-7cf03711ed12)

**Comment by DKeken** [comment]
`{
	"compilerOptions": {
		"module": "commonjs",
		"declaration": true,
		"incremental": false,
		"declarationMap": true,
		"noImplicitAny": false,
		"importHelpers": true,
		"esModuleInterop": true,
		"removeComments": false,
		"noLib": false,
		"forceConsistentCasingInFileNames": true,
		"emitDecoratorMetadata": true,
		"experimentalDecorators": true,
		"target": "ES2021",
		"sourceMap": true,
		"skipLibCheck": true,
		"resolveJsonModule": true,
		"strict": true,
		"outDir": "./dist",
		"rootDir": "./src"
	},
	"exclude": ["node_modules", "dist", "src/**/__tests__/*", "*.spec.ts"]
}`

**Comment by KevinEdry** [comment]
Thanks for opening an issue, thats a duplicate of: #28 
I'll push a fix today for it and will update this issuue.

**Comment by KevinEdry** [comment]
Solved in `v1.5.2`, please confirm before closing the issue.
🔗 https://github.com/KevinEdry/nestjs-trpc/releases/tag/1.5.2
🔗 https://www.npmjs.com/package/nestjs-trpc
